### PR TITLE
Replace unsupported :host-context with :dir

### PR DESCRIPTION
**Pull request by awikkerink**
Tested on Chrome, Firefox, IE11

### CODE REVIEW

**Comment by awikkerink** [comment]
Working combinations:
LTR + Shadow + Native CSS
LTR + Shady + Native CSS
LTR + Shady + Shim CSS
RTL + Shady + Native CSS
RTL + Shady + Shim CSS

Not working:
LTR + Shadow + Shim CSS
RTL + Shadow + Native CSS
RTL + Shadow + Shim CSS

Conclusion: Don't use Shadow DOM yet

**Comment by awikkerink** [comment]
`:host-context` doesn't work with Shady + Shim CSS. So IE11/Edge don't work with `:host-context`, but do work with `:dir`.

File: d2l-offscreen-shared-styles.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 					left: -10000px;
 				};
 			}
-			:host-context([dir="rtl"]) {
+			:host(:dir(rtl)) {
```
Comment (dlockhart) [inline d2l-offscreen-shared-styles.html]: Hmm... I'm really surprised by this (thought we would have tested shady A LOT). Also, caniuse says the `:dir` pseudo class shouldn't be working in Chrome/IE/Edge/Safari:
http://caniuse.com/#feat=css-dir-pseudo

Comment (awikkerink) [inline d2l-offscreen-shared-styles.html]: https://github.com/webcomponents/shadycss/blob/c253c54860b743e680f4a4cc2262b1ab71a292ef/src/style-transformer.js#L32

**Pull request merged**